src/connection_string.ts

```diff
@@ -33,6 +33,12 @@ import { Logger, LoggerLevel } from './logger';
 import { PromiseProvider } from './promise_provider';
 import { Encrypter } from './encrypter';
 
+const VALID_TXT_RECORDS = ['authSource', 'replicaSet', 'loadBalanced'];
+
+const LB_SINGLE_HOST = 'loadBalanced option only supported with a single host in the URI';
```

Comment:
nit: add a suffix indicating this is an error string (maybe `LB_SINGLE_HOST_ERROR`), also consider inlining since it doesn't seem to be used in multiple places

Reply:
Updated.